nixos/tests/mysql/mysql.nix

```diff
@@ -185,7 +185,6 @@ import ./../make-test-python.nix ({ pkgs, ...} : {
     mariadb.succeed(
         "echo 'use testdb; drop table rocksdb;' | sudo -u testuser mysql -u testuser"
     )
-  '' + pkgs.stdenv.lib.optionalString pkgs.stdenv.isx86_64 ''
```

Comment:
readd this line

Reply:
Assuming the following tests are x86_64, you still need this line so they don't run on e.g. aarch64

Reply:
Sorry I didn't read your initial comment correctly. It should be fine now